nginx-nodejs-redis/docker-compose.yml

```
@@ -0,0 +1,25 @@
+
+services:
+  redis:
+    image: 'redislabs/redismod'
+    ports:
+      - '6379:6379'
+  web1:
+    restart: on-failure
+    build: ./web1
```

Use the `web` directory to build the image and setup a dedicated `hostname`
```suggestion
  web1:
    restart: on-failure
    build: ./web
    hostname: web1
```